AardvarkTests/ARKDataArchiveTests.m

```diff
@@ -95,7 +95,7 @@ - (void)test_setUp_providesEmptyArchive;
         [expectation fulfill];
     }];
     
-    [self waitForExpectationsWithTimeout:1.0 handler:nil];
+    [self waitForExpectationsWithTimeout:5.0 handler:nil];
```

Comment:
Tests were slightly flaky when running in parallel on a slow machine with a 1 second timeout, so I bumped all tests to 5 seconds. (tried 2 seconds, but Travis CI still error'd out)